index.html

```diff
@@ -1,48 +1,122 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title> Golpes Básicos </title>
+        <title> Bases del Boxeo </title>
         <meta charset="UTF-8">
-        <meta name="description" content="Aprende los golpes basicos del deporte">
+        <meta name="description" content="Aprende las bases del ">
+        <link rel="stylesheet" href="style/style.css">
     </head> 
-    <body style="background-color: darkslategrey;">
+    <body>
        <header>
-           <h1 style="color: maroon;"> El Boxeo</h1>
+           <h1 class="titles"> El Boxeo</h1>
            <hr>
+           <h1 class="subtitulo">Golpes</h1>
            <h2><p>El boxeo es un deporte de contacto en donde dos contrincantes se enfretan solamente usando sus puños</p></h2>
            <h2><p>Al solo utilizarse los puños la cantidad de golpes posibles son limitados. Es por eso que es facil listarlos</p> </h2>
-           <h2><p>A continuacion se encuentran los golpes mas importantes y remarcables del boxeo con su respectiva demostración y descripción</p></h2>           
+           <h2><p>A continuacion se encuentran los golpes mas importantes y remarcables del boxeo con su respectiva demostración y descripción</p></h2>
```

Comment:
It's not necessary to add a `<p>` when you have a `<h2>` tag. You can remove it and it'll work just fine